discussment-examples/forum/src/main/java/org/danekja/discussment/forum/core/service/DashboardService.java

```diff
@@ -0,0 +1,30 @@
+package org.danekja.discussment.forum.core.service;
+
+import org.danekja.discussment.core.domain.Category;
+import org.danekja.discussment.core.domain.Discussion;
+import org.danekja.discussment.core.domain.Topic;
+import org.danekja.discussment.forum.core.domain.User;
+
+public interface DashboardService {
```

Comment:
See comment for DashboardService in article example. Remove this.

Note: if you need to copy and paste this much code, it usually means something is not done correctly.